react/components/EXPERIMENTAL_Table/Sections/index.tsx

```diff
@@ -76,5 +78,6 @@ const FowardedSections: ComposableSections = forwardRef(Sections)
 
 FowardedSections.Head = Thead
 FowardedSections.Body = Tbody
+FowardedSections.Hideable = LoadedView
```

Comment:
Why hideable and not loaded?

Reply:
It was a "not saved" issue, fixed now :smile: